code/modules/events/immovable_rod.dm

```diff
@@ -27,7 +27,7 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	announceWhen = 5
 
 /datum/round_event/immovable_rod/announce(fake)
-	priority_announce("What the fuck was that?!", "General Alert")
+	priority_announce("Unknown object detected on collision course with the station.", "General Alert")

```

Comment:
If anyone has a better suggestion for specific text, feel free of course

Reply:
"Unknown object detected on collision co-" CLANG "What the fuck was that?!"

Reply:
something like that could definitely work, we can brainstorm more ideas for the absolute best version